src/content/docs/en/recipes/build-forms-api.mdx

```diff
@@ -327,52 +294,136 @@ This recipe shows you how to send form data to an API endpoint and handle that d
+        <script lang="ts">
+          let responseMessage: string;
+
+          async function submit(e: SubmitEvent) {
+            e.preventDefault();
+            const formData = new FormData(e.currentTarget as HTMLFormElement);
+            const response = await fetch("/api/feedback", {
+              method: "POST",
+              body: formData,
+            });
+            const data = await response.json();
+            responseMessage = data.message;
+          }
+        </script>
+
+        <form on:submit={submit}>
+          <label>
+            Name
+            <input type="text" id="name" name="name" required />
+          </label>
+          <label>
+            Email
+            <input type="email" id="email" name="email" required />
+          </label>
+          <label>
+            Message
+            <textarea id="message" name="message" required />
+          </label>
+          <button>Send</button>
+          {#if responseMessage}
+            <p>{responseMessage}</p>
+          {/if}
+        </form>
             ```
-          </Fragment>
+      </Fragment>
+      <Fragment slot="vue">
+        ```vue title="src/components/FeedbackForm.vue" "/api/feedback" add={1-16, 33} "formData" "e.preventDefault();"
+        <script setup lang="ts">
+        import { ref } from "vue";
+
+        const responseMessage = ref<string>();
+
+        async function submit(e: Event) {
+          e.preventDefault();
+          const formData = new FormData(e.currentTarget as HTMLFormElement);
+          const response = await fetch("/api/feedback", {
+            method: "POST",
+            body: formData,
+          });
+          const data = await response.json();
+          responseMessage.value = data.message;
+        }
+        </script>
 
-        </UIFrameworkTabs>
+        <template>
+          <form @submit="submit">
+            <label>
+              Name
+              <input type="text" id="name" name="name" required />
+            </label>
+            <label>
+              Email
+              <input type="email" id="email" name="email" required />
+            </label>
+            <label>
+              Message
+              <textarea id="message" name="message" required />
+            </label>
+            <button>Send</button>
+            <p v-if="responseMessage">{{ responseMessage }}</p>
+          </form>
+        </template>
+        ```
+      </Fragment>
+
+    </UIFrameworkTabs>
+
+4. Import and include your `<FeedbackForm />` component on a page. Be sure to use a `client:*` directive to ensure that the form logic is hydrated when you want it to be.
+    
+    <UIFrameworkTabs>
+        <Fragment slot="preact">
+        ```astro title="src/pages/index.astro" "client:load"
+        ---
+        import FeedbackForm from "../components/FeedbackForm"
+        ---
```

Comment:
```suggestion
        ---
        export const prerender = false;
        import FeedbackForm from "../components/FeedbackForm"
        ---
```

Reply:
@sarah11918 if preferred, I can add this to the code snippet at the end as well.

Reply:
Ah, I guess the *page itself* doesn't need to be on-demand rendered as long as the API endpoint is. So, in fact, in hybrid mode, shouldn't `feedback.ts` need `export const prerender = false;`?

Reply:
SO, if having the statement doesn't actually make a difference, let's not put it in and... see what happens!  :grinning: